### ISSUE TITLE
[1.3 beta 5] processing errors

### ISSUE THREAD

**Issue by timothyarmes** [meteor/meteor#6073]
I'm trying to update from beta 4 to beta 5, however as a result of the my project is failing to process 2 packages.

First error is this: 

```
packages/kadira:flow-router-ssr/lib/router.js:43:8: /packages/kadira:flow-router-ssr/lib/router.js: Duplicate declaration "path"
```

Which fails on this code:

```
// XXX this function needs to be cleaned up if possible by removing `if (this.isServer)`
  // and `if (this.isClient)` if possible
  path(pathDef, fields, queryParams) {
    if (this._routesMap[pathDef]) {
      pathDef = this._routesMap[pathDef].path;
    }

    let path = '';
```

Presumably it no longer likes the fact `let path`has the same name as the function itself?

Second error:

```
packages/kadira:flow-router-ssr/client/router.js: /packages/kadira:flow-router-ssr/client/router.js: "route" is read-only (This is an error on an internal node. Probably an internal error. Location has been estimated.)
```

For which we have no further info.

Third error:

```
packages/ultimatejs:tracker-react/tracker-react-mixin.js:2:21: /packages/ultimatejs:tracker-react/tracker-react-mixin.js: 'return' outside of function (2:21)
```

The first two lines of the file are:

```
TrackerReact = {}; //so server side rendering isn't broken
if (Meteor.isServer) return;
```

In this case the error makes sense to me, however I don't understand how it ever worked in the first place?

**Comment by jasongrishkoff** [comment]
I'm encountering this same issue after upgrading to beta5.

**Comment by SylarRuby** [comment]
Any update on this?

**Comment by timothyarmes** [comment]
It looks like updating Babel has caused various issues. My concern is that this isn't something the Meteor team can fix :(

**Comment by arunoda** [comment]
This is is totally valid code and not sure what went wrong. @timothyarmes I don't think it's a babel error. I used babel6 in many projects it works pretty well.

When I read sometimes back, Meteor picks babel rules manually rather using babel directly. I hope this is something happen in the upgrading process. Not something in babel itself.

**Comment by timothyarmes** [comment]
@arunoda I agree that the code is valid. Are you suggesting that this only happens when upgrading? You you able to reproduce the problem.

At this point I still can't run my project using beta6.

**Comment by benjamn** [comment]
The "Duplicate declaration" errors are being addressed in #6138.

The "return outside of function" error is being addressed in #6144.

**Comment by avital** [comment]
(And on a meta-note; please don't open issues that refer to more than one bug -- we nee to be able to track each one separately)
